examples/api/src/index.ts

```diff
@@ -205,7 +206,11 @@ async function asyncMain() {
     )
   }
 
-  if (process.env.PAYREXX_INSTANCE_NAME && process.env.PAYREXX_API_SECRET) {
+  if (
+    process.env.PAYREXX_INSTANCE_NAME &&
+    process.env.PAYREXX_API_SECRET &&
+    process.env.PAYREXX_WEBHOOK_SECRET
+  ) {
     paymentProviders.push(
       new PayrexxPaymentProvider({
```

Comment:
Since no PAYREXX_WEBHOOK_SECRET is required for PayrexxSubscription provider only, we should probably differentiate it, so that one can add only one of the provider OR also both.

Reply:
Since this i just an example i would let it like this...